sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/customer/_manager.py

```diff
@@ -189,6 +189,7 @@ def _do_initialize(self, connection_string: str) -> bool:
 
             # Set status to active after successful initialization
             self._status = CustomerSdkStatsStatus.ACTIVE
+            set_statsbeat_customer_sdkstats_feature_set()
```

Comment:
Let's be consistent with [live metrics](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_live_metrics.py#L49) and set this in the global [api](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/customer/_customer_sdkstats.py#L13) for customer stats